File: modules/core/server-scala/src/main/scala/modules/jwt/model/Tokens.scala
package modules.jwt.model

import spray.json.{DefaultJsonProtocol, RootJsonFormat}

case class Tokens(accessToken: String, refreshToken: String)

object Tokens extends DefaultJsonProtocol {
  implicit val tokensFormat: RootJsonFormat[Tokens] = jsonFormat2(Tokens.apply)
}

File: modules/authentication/server-java/src/main/java/com/sysgears/authentication/model/jwt/Tokens.java
package com.sysgears.authentication.model.jwt;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Tokens {
    private String accessToken;
    private String refreshToken;
}
